src/main/java/forestry/apiculture/items/ItemScoop.java

```diff
@@ -49,4 +61,20 @@ public boolean mineBlock(ItemStack stack, Level world, BlockState blockState, Bl
 
 		return true;
 	}
+
+	@Override
+	public InteractionResult interactLivingEntity(ItemStack stack, Player player, LivingEntity interactionTarget, InteractionHand usedHand) {
+		if(!interactionTarget.level().isClientSide()) {
+			if (interactionTarget instanceof Bee) {
```

Comment:
The order is messed up. Check if it's instanceof Bee, then put the isClientSide check. Only have two returns- one inside the instanceof Bee that returns sidedSuccess, and one outside that returns PASS when it's not a bee. The idea here is, you check if the entity is a bee, and if it is, run the game logic on the server, and tell the client to swing their arm by returning sidedSuccess.